wrapper/CSharp/wolfSSL-TLS-Server/wolfSSL-TLS-Server.cs

```diff
@@ -222,6 +233,37 @@ public static void Main(string[] args)
         }
         Console.WriteLine(buff);
 
+        /* get and print sni from a sample buffer, can be used by using the raw client hello */
+        if (haveSNI(args)) {
+            IntPtr result = Marshal.AllocHGlobal(32);
+            IntPtr inOutSz = Marshal.AllocHGlobal(sizeof(int));
+            Marshal.WriteInt32(inOutSz, 32);
+            byte [] buffer = { /* www.paypal.com */

```

Comment:
Please avoid the full link, just say from PayPal website client hello. Is there a reason you didn't use our own or something not money related like google.com?